src/styles/_stars.scss

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+  }
+
+  &--1 > &__star:first-child {
+    background-image: url(/images/star-active.svg);
+  }
+  &--2 > &__star:nth-of-type(-n + 2) {
+    background-image: url(/images/star-active.svg);
+  }
+  &--3 > &__star:nth-of-type(-n + 3) {
+    background-image: url(/images/star-active.svg);
+  }
+  &--4 > &__star:nth-of-type(-n + 4) {
+    background-image: url(/images/star-active.svg);
+  }
+  &--5 > &__star:nth-of-type(-n + 5) {
+    background-image: url(/images/star-active.svg);
```

Comment:
Try to combine these selectors so as not to duplicate so many times `background-image`